lib/embed/embed.dart

```diff
@@ -96,6 +96,28 @@ class PlaygroundMobile {
     });
   }
 
+  /**
+   * Return if strong mode should be enabled
+   */
+  bool _isStrongMode(String strong) {
+    if (strong == null) return false;
+    return !(strong == 'false' || strong == 'f');
```

Comment:
I'd prefer a test for whether it's true, rather than an inverted test for not-false.